util.js

```diff
@@ -33,8 +33,44 @@ debugerror.log = console.error.bind(console)
 /** @function */
 const isBinaryFileAsync = module.exports.isBinaryFileAsync = Promise.promisify(require('isbinaryfile'))
 
+function removePassword (input) {
+  return input.replace(/(-P |pass:|\/p|-pass )([^ ]+)/, function (match, p1, p2) {
+    return `${p1}***`
+  })
+}
+
 /** @function */
-module.exports.execFileAsync = Promise.promisify(child.execFile)
+module.exports.execFileAsync = function (file, args, options) {
+  if (debug.enabled) {
+    debug(`Executing ${file} ${args == null ? '' : removePassword(args.join(' '))}`)
+  }
+
+  return new Promise((resolve, reject) => {
```

Comment:
Both electron-packager and electron-builder require nodejs 4+. So, I think, it is ok to use template strings.